genny/assets/webpack/templates/assets/js/application.js.tmpl

```diff
@@ -1,5 +1,6 @@
 require("expose-loader?$!expose-loader?jQuery!jquery");
 require("bootstrap/dist/js/bootstrap.bundle.js");
+require("@fortawesome/fontawesome-free/js/all.js");
```

Comment:
Should this be `@fontawesome` instead?

Reply:
no for whatever reason they renamed it...  I did a double take the first time as well.  Such is the life of front end development righ?

Reply:
I confirm what @corylanou says:
https://www.npmjs.com/package/@fortawesome/fontawesome-free